text/000x-nrd-kernels.md

```diff
@@ -0,0 +1,332 @@
+
+- Title: nrd-kernels
+- Authors: [Antioch Peverell](mailto:apeverell@protonmail.com)
+- Start date: Mar 24, 2020
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000)
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+[summary]: #summary
+
+Grin supports a limited implementation of "relative timelocks" with "No Recent Duplicate" (NRD) transaction kernels. Transactions can be constructed such that they share duplicate kernels. An NRD kernel instance is not valid within a specified number of blocks relative to a prior duplicate instance of the kernel. A minimum number of blocks must therefore exist between two instances of an NRD kernel. This provides a relative timelock between transactions.
```

Comment:
"A minimum number of blocks" -> "A minimum height difference"

this avoid the ambiguity in how to count in between blocks